commands/channels/count.mjs

```diff
@@ -66,6 +73,56 @@ const getPinsLeaderboard = async function(client) {
     return await buildLeaderboardEmbed(leaderboard, 'pins')
 }
 
+const cleanMessages = async function(client, commit) {
```

Comment:
I recommend we delete the leaderboard.json at the start of this function. That way after it cleans the next time the count command is called it recounts everything

Reply:
I'm not sure how the leaderboard.json get saved. Doesn't this call always count from the beginning since I'm not passing in any existing data?

Reply:
Nvm. It now removes leaderboard.json when the messages are deleted.